profiles/Amandeep.json

```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1227#issue-1303348005",
+    "issueId": 1227,
+    "name": "Amandeep Singh",
+    "username": "Amandeep404"

```

Comment:
The indentation seems to be wrong here.
```suggestion
  "image": "https://github.com/EddieHubCommunity/awesome-github-profiles/issues/1227#issue-1303348005",
  "issueId": 1227,
  "name": "Amandeep Singh",
  "username": "Amandeep404"
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Inefficient